alerting/src/main/resources/org/opensearch/alerting/alerts/finding_mapping.json

```diff
@@ -51,6 +51,9 @@
     },
     "timestamp": {
       "type": "long"
+    },
+    "execution_id": {
+      "type": "keyword"
```

Comment:
We need to increment the schema version whenever we update the schema for a release.